lib/portage/util/portage_lru_cache.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+import os
+import portage
+
+
+def show_lru_cache_info():
+    if not os.environ.get("PORTAGE_SHOW_LRU_CACHE_INFO"):
+        return
```

Comment:
Maybe make this a FEATURES setting instead of environment variable?

Reply:
I considered that. But it feels more like a debug thing that can be enabled via an env var, than a user facing env var. That is, I feel like to should be more hidden from users.That said, I wouldn't oppose a FEATURE if there is consensus that it's the right thing to do.

Reply:
> I considered that. But it feels more like a debug thing, that can be enabled via an env var, than a user facing FEATURE setting. I feel like to should be more hidden from users.That said, I wouldn't oppose a FEATURE if there is consensus that it's the right thing to do.

Reply:
I don't think this makes sense as a FEATURES. That implies that users are supposed to report whether it's set as part of e.g. attaching `emerge --info` to a bug report... it also implies that users are expected to set this in make.conf so it prints on every run, rather than running it as part of development profiling.

Reply:
Okay, I agree that FEATURES is not a good fit for this.